.github/workflows/root-ci-config/buildconfig/alma9.txt

```diff
@@ -1,4 +1,4 @@
 builtin_nlohmannjson=ON
 builtin_vdt=On
 tmva-sofie=On
-tmva-cpu=OFF
+BLA_VENDOR=OpenBLAS
```

Comment:
What is this option doing? If it exists, shouldn't it be `BLAS_VENDOR`?

Reply:
It's a funny name, yes :D But that's how it is:
https://cmake.org/cmake/help/latest/module/FindBLAS.html#input-variables

It forces the use of openblas instead of flexiblas. More context here:
https://github.com/root-project/root/issues/15309#issuecomment-2097506361

Reply:
Looks like the new default blas in alma9 (flexiblas) can't be used simultaneously with the openblas that is builtin to NumPy. But using OpenBlas is also a good configuration, since this is also what the LCG packages use I think